### ISSUE TITLE
CLI Command: "landscaper init"

### ISSUE THREAD

**Issue by In-Ko** [gardener/potter-hub#3]
**Description**
The Landscaper CLI needs to provide an easy way to enable local development scenarios. This command should include the landscaper installation itself (into an already existing cluster, this is a prerequisite), and the installation of a locally running OCI Registry, which should be wired to the landscaper as default OCI Registry.

**Done Criteria**
- [ ] Concept for "landscaper init" created, discussed and agreed
- [ ] All needed Features implemented
- [ ] Code has been reviewed by other team members
- [ ] Unit Tests created for new code
- [ ] Integration Test Suite updated (if applicable)
- [ ] Internal technical Documentation updated
- [ ] Enduser Documentation updated (if applicable)
- [ ] Successful demonstration in Review

**Comment by In-Ko** [comment]
Please also check out / take into account the work done here:
- https://github.com/dav9/landscaper/tree/landscaper-qs.
- The command is here: https://github.com/dav9/landscaper/tree/landscaper-qs/cmd/landscaper-qs
- Build it with: https://github.com/dav9/landscaper/blob/landscaper-qs/hack/install-dev.sh
- You need cluster with ingress enabled, because of harbor.
- Then you export kubeconfig= ….and you do
```
landscaper-qs install-landscaper
landscaper-qs add-definition
landscaper-qs add-instance
```

**Comment by loewenstein** [comment]
Should this support local k8s clusters like KinD and minikube? If so, is it worth mentioning that explicitly and probably add tests that ensure it.

Not sure if the usual differences like no load balancer type services are relevant at all for the landscaper...

**Comment by In-Ko** [comment]
Yes, kind and minikube should be supported. In the end, it should not really matter what kind of k8s cluster you are working with in this local dev scenario, that is why it is not mentioned explicitly above.
